config/owasp/suppressions.xml

```diff
@@ -8,4 +8,12 @@
         ]]></notes>
         <cve>CVE-2018-1258</cve>
     </suppress>
+    <suppress until="2021-01-05">
+        <notes><![CDATA[
+   file name: oauth2-oidc-sdk-7.1.1.jar
```

Comment:
Have we been given the go ahead to suppress this?

Reply:
Yes @AdilOoze